python/GafferImageUI/ImageWriterUI.py

```diff
@@ -487,7 +487,11 @@ def __extension( parent ) :
 
 			"description",
 			"""
-			The data type to be written to the OpenEXR file.
+			The data type to be written to the OpenEXR file.  If you want to use different
+			data types for different channels, you can drive this with an expression or spreadsheet,
+			which may use the same context variables as the layout plugs ( the useful ones are
+			`${imageWriter:channelName}`, `${imageWriter:layerName}` and `${imageWriter:baseName}`,
+			for the whole channel name, and for the prefix and suffix respectively ).
```

Comment:
If we add this extra metadata, then the "Create Spreadsheet" menu item will automatically populate the selector for us :

```"ui:spreadsheet:selectorValue", "${imageWriter:channelName}"```